ingeniamotion/wizard_tests/base_test.py

```diff
@@ -42,6 +43,13 @@ def save_backup_registers(self) -> None:
             except IMRegisterNotExist as e:
                 self.logger.warning(e, axis=self.axis)
 
+        if not isinstance(self.optional_backup_registers_names, List):
```

Comment:
Why do you do that? Can optional_backup_registers_names not be a list?

Reply:
Moreover, I think `List` (with uppercase) should be used only for typing.